### DISCUSSION TITLE
Type of body parameters - array

### DISCUSSION THREAD

**Discussion by MarinaLivshits24** [GitbookIO/community#222] [4. Open API]
### Ask a question, start a conversation, or make an annoucement

We have yaml and json file with API requests and we import it into GitBook.
We have problem with showing in GitBook the type of parameters array.
In swagger it is shown correctly: array[string] (query), but in GitBook it is shown just like string.

Is it a bug? Do you have any ideas how to fix it?

**Comment by addisonschultz** [comment]
Hi @MarinaLivshits24 !

Can you share an image/example of how it's importing? Additionally, can you provide more context on how you are importing these files?
